main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1059,6 +1059,12 @@ object Resolver {
         case (e, rs) => ResolvedAst.Expr.RestrictableChoose(star, e, rs, loc)
       }
 
+    case NamedAst.Expr.ExtensibleMatch(label, exp1, sym2, exp2, sym3, exp3, loc) =>
+      mapN(resolveExp(exp1, scp0), resolveExp(exp2, scp0 ++ mkVarScp(sym2)), resolveExp(exp3, scp0 ++ mkVarScp(sym3))) {
```

Comment:
Easier read if we extract these to variables.

Reply:
Then I have to name them- increasing the risk of errors.

Reply:
Readability affects error rate, too.